templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Car</h1>
+<p>Are you relly want to delete car: <strong class="larger-text">{{ car }}</strong> ?</p>
+
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger" >
+  </form>
+
+      <a
+        href="{% url 'taxi:car-detail' pk=car.id %}"
+    > <input class="btn btn-primary" type="submit" value="CANCEL">

```

Comment:
Do not break the tag in this way, place arguments on the same line with that tag.